docs/index.mdx

```diff
@@ -1,36 +1,42 @@
+<Card title="🎉 Widgetbook Cloud is now Live!" href="/cloud">
+  **Save 20% of your time** by shortening the feedback loop between designers
+  and developers.
+</Card>
+
 # Overview
 
-Discover Widgetbook, a powerful Flutter package inspired by Storybook.js that simplifies the process of cataloging widgets, testing them across various devices and themes, and sharing them effortlessly with designers and clients.
+Widgetbook is a Flutter package inspired by [Storybook.js](https://storybook.js.org/) that simplifies the process of cataloging widgets to create your own [design system](/glossary/design-system), and testing the widgets across various configurations, like different devices, languages and themes.
+
+Widgetbook is [recommended by Google's Flutter team](https://www.youtube.com/watch?v=-g2KYjiPzzA) who features us as experts for design systems and designer/developer collaboration.
 
 <Image src="/assets/screenshots/widgetbook.png" zoom />
 
-## Experience Widgetbook in Action
+## The Problem
+
+Developing user interfaces in Flutter applications often involves creating and managing numerous widgets. However, developers face several challenges in the process:
+
+- **Keeping the overview of the design system is difficult** Setting up and maintaining the structure for a [design system](/glossary/design-system) that catalogs every widget is time-consuming. As it's not the core business of a Flutter team, it also distracts the developers from actual feature development.
+
+- **UI flaws:** Testing widgets across multiple devices, themes, and settings is time-consuming and prone to errors. When changing existing widgets, it is difficult to detect all resulting UI changes to ensure that all changes look and feel as intended.
```

Comment:
What about the collaboration problem between designers and developers?  I'd feature that problem here as well. So that we can feature Widgetbook Cloud as a solution that we can then cross reference